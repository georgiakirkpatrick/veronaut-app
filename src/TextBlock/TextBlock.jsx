import React from 'react'
import './TextBlock.css'

const TextBlock = props => {
  const {
    text
  } = props
  
  return <div className='TextBlock'>
    {text}
  </div>
}

TextBlock.defaultProps = {
  text: ''
}

export default TextBlock