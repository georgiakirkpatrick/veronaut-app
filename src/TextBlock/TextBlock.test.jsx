import React from 'react'
import { render, screen } from '@testing-library/react'
import { describe, it, expect } from 'vitest'
import TextBlock from './TextBlock'

describe ('TextBlock', () => {
  it ('renders without crashing', () => {
    render(<TextBlock />)
    screen.debug()
  })

  it('renders the UI as expected', () => {
    const tree = render(<TextBlock />)
    expect(tree).toMatchSnapshot()    
  })
})