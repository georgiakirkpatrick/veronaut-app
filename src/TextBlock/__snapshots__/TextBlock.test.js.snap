// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextBlock renders the UI as expected 1`] = `
<div
  className="TextBlock"
>
  
</div>
`;
